scripts/get_golang_version.sh

```diff
@@ -1,2 +1,16 @@
 #!/usr/bin/env bash
-awk '/^go[ \t]+[0-9]+\.[0-9]+(\.[0-9]+)?[ \t]*$/{print $2}' go.mod
+# Required GOLANG Version for project specified here
+GOLANG_VERSION="1.12.17"
+
+# Check to make sure that it matches what is specified in go.mod
+
+GOMOD_VERSION=$(awk '/^go[ \t]+[0-9]+\.[0-9]+(\.[0-9]+)?[ \t]*$/{print $2}' go.mod)
+TRIMED_GOLANG_VERSION=$(echo ${GOLANG_VERSION} |awk -F. '{ print $1 "." $2 }')
```

Comment:
You have a typo here `TRIMED` -> `TRIMMED`.

Once that's fixed and the merge conflict resolved I can approve it.

Reply:
Good catch, updated.

Reply:
Thanks